scripts/fuzz_opt.py

```diff
@@ -115,10 +118,18 @@ def randomize_feature_opts():
     print('randomized feature opts:', ' '.join(FEATURE_OPTS))
 
 
+# the optimizations to run on the wasm
 FUZZ_OPTS = None
+# whether NaN values are allowed, or we de-NaN them
```

Comment:
I agree this isn't great. But `None` is meant to indicate a null value. It will always be replaced later. So I worry that setting to `False` or `True`, while technically valid, could be confusing as it could indicate a default, but it isn't that.

Another option is to not declare these in the global scope at all, and move the comments to where these are assigned. I'll push a commit with that, let me know what you think.

Reply:
Oh, I see. It might be good to reword the comments to be more explicit about what values will eventually be assigned to these globals, since right now they sound like plain Boolean options.